CalendarPicker/Day.js

```diff
@@ -41,12 +41,21 @@ export default function Day(props) {
     disabledDatesTextStyle,
     minRangeDuration,
     maxRangeDuration,
-    enableDateChange
+    enableDateChange,
+    renderDay
   } = props;
 
   const thisDay = new Date(year, month, day, 12);
   const today = new Date();
 
+  const DayComponent = ({ style }) => {
+    if (renderDay) {
+      return renderDay({ year, month, day, date: thisDay.format('YYYY-MM-DD'), style });
```

Comment:
recommend returning the Date object instead of formatting it. Different users may use different formats, and this forces everyone to use a single format that may not match.

Reply:
Good idea